submissions/maxim-zabolotny/HTML-CSS-Popup/index.html

```diff
@@ -0,0 +1,122 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
+  <title>Popup window</title>
+</head>
+<body>
+  <header>
+    <nav>
+      <ul class="user_menu">
+        <li class="user_menu__element user_menu__element-text" >
+          <a href="#">Mail</a>
+        </li>
+        <li class="user_menu__element user_menu__element-text">
+          <a href="#">Images</a>
+        </li>
+        <li>
+          <input type="checkbox" class="apps_menu__checkbox" id="apps_checkbox">
+          <label for="apps_checkbox" class="user_menu__element user_menu__element-anchor">
+            <img src="img/popup-button.png" alt="apps">
+          </label>
+          <div class="apps_menu">
+            <div class="apps_menu__content">
+              <div class="first_page">
+                <a href="#" class="icon_card">
+                  <img src="img/calendar.png" alt="Calendar">
+                  Calendar
+                </a>
+                <a href="#" class="icon_card">
```

Comment:
I recommend replace your `div` on `ul` or `ol` and wrap your links (each separately) in `li`.
All it for semantics

Reply:
All fixed
Thanks for your help